flight-icons/README.md

```diff
@@ -0,0 +1,35 @@
+# flight-icons
+
+An addon for the icons from `flight`. Currently in development. This addon can be used in React or Ember apps.
+
+## Installation
+
+```bash
+yarn add @hashicorp/flight-icons
+```
+
+## Usage
+
+We intend for icons to be used at a 16 or 24 `viewBox` size.
+
+### How to choose 16 or 24 `viewBox` size
+
+* If an icon is in a `viewBox` <= 16, use the `-16` version of the icon.
```

Comment:
issue: while reading, I am not sure what to do with this information, is not clear for me (as a developer)

I am not 100% sure about what we want to say here (also in relation to the second part below), but a few thoughts while reading it:
- the word "viewBox" reminds me of the SVG "viewBox" property, but from what I understand the icons can be used in different ways, right? e.g. in _react-components_ icons are used via `<InlineSvg src={svgFile} />` so in that case I don't think the size is set via the viewBox, right?
- here we say that for intermediate sizes (16 < x < 24) always use 16, but below we have a different criteria; I think the two are kinda antithetic? I suspect we should give the same advice (probably even merge them, as suggested below)